src/main/java/uk/org/tombolo/importer/ons/ONSWagesImporter.java

```diff
@@ -138,6 +139,11 @@ public DatasourceSpec getDatasourceSpec(String datasourceIdString) throws Except
         return DatasourceId.valueOf(datasourceIdString).datasourceSpec;
     }
 
+    @Override
+    protected List<String> getOaDatasourceIds() {
+        return Collections.singletonList(OaImporter.OaType.localAuthority.datasourceSpec.getId());
+    }
+
```

Comment:
I believe this would restrict user to work only with localAuthority boundaries

Reply:
Please see the general comment.